<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDialog</class>
 <widget class="QDialog" name="ControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View Control Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>271</width>
     <height>83</height>
    </rect>
   </property>
   <property name="title">
    <string>Trajectory Integration in cavity (W)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>No steps / wavelength</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Accuracy</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBoxNoSteps">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="ScientificDoubleSpinBox" name="techSpinBoxAccuracy"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Global Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Symmetry</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="radioButtonRing">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Rin&amp;g</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonLine">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Line</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxUseClosedOrbit">
      <property name="text">
       <string>Use excited orbit as reference</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>271</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>View Menu Function Preferences</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxUseTotalSize">
      <property name="text">
       <string>Include dispersion contribution in rms size</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxRewrite">
      <property name="text">
       <string>Rewrite text buffer when analyzing </string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxDrawApertures">
      <property name="text">
       <string>Draw apertures on beam size plots</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QSpinBox" name="spinBoxPlotSamples">
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="singleStep">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxClearPlot">
      <property name="text">
       <string>Clear plots</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Coupling Threshold </string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>No of samples in plots</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxAutoPrint">
      <property name="text">
       <string>Interactive lattice function tables</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxClearText">
      <property name="text">
       <string>Clear text</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxAutoPlot">
      <property name="text">
       <string>Interactive lattice function plots</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBoxDrawLego">
      <property name="text">
       <string>Draw Lego Plots</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="ScientificDoubleSpinBox" name="techSpinBoxCoupling"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ScientificDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ControlDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ControlDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
